src/applications/disability-benefits/2346/config/form.js

```diff
@@ -115,6 +115,10 @@ const formConfig = {
   verifyRequiredPrefill: true,
   introduction: IntroductionPage,
   confirmation: ConfirmationPage,
+  reviewPageTitle: 'Review order details',
+  saveSuccessMessage: 'Order has been saved.',
+  startNewAppMessage: 'Start a new order',
+  continueAppMessage: 'Continue your order',
   footerContent: FormFooter,
```

Comment:
It might be nice to have these custom messages grouped together, something like `customMessages`, instead of having them all as separate keys in the config/form root. This is just a personal opinion.

Reply:
Interesting. Maybe I could refactor the savedFormMessages to hold all custom messages?

Reply:
@cvalarida @Mottie Let me know what you all think of the refactor.